cloudflared/rootfs/etc/services.d/cloudflared/run

```diff
@@ -14,10 +14,17 @@ else
     if bashio::config.has_value 'data_folder'; then
         data_path="/$(bashio::config 'data_folder')/cloudflared"
     fi
+    # Check status file which exists when custom_config exists
+    if bashio::fs.file_exists "/tmp/custom_config"; then
```

Comment:
We don't need that if we go with the additional config flag. Otherwise I would still remove this and check for `if bashio::fs.file_exists "${data_path}/config.yml" ; then`. Of course this also only works, if we implement the fail above if the file is there but not valid as suggested by me.

Reply:
Also relevant on other locations